sw/airborne/arch/chibios/STM32F405xG.ld

```diff
@@ -20,7 +20,7 @@
  */
 MEMORY
 {
-    flash0  : org = 0x08004000, len = 1M
+    flash0  : org = 0x08010000, len = 1M
```

Comment:
Both the `crazyflie_2.1` and the `lisa_mxs` use this file. Are you sure this is correct and they both have a bootloader at the first part of the flash of this size?